### PR TITLE
docs: better inform message for dm rows (#414)

### LINKED ISSUE #414: better inform() text for dm_rows()
Current messaging when the in_place argument is missing:

`message("Not persisting, use `in_place = FALSE` to turn off this message.")`

More consistent would be to use similar text to dm::target_table_name():

`inform("Result is returned as lazy table. Use `in_place = FALSE` to mute this message, or `in_place = TRUE` to write to the underlying table.")`

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Closes #414

### CODE REVIEW

File: R/rows-dm.R [hunk 1]
```diff
@@ -206,7 +206,7 @@ dm_rows <- function(x, y, operation_name, top_down, in_place, require_keys, prog
   dm_rows_check(x, y)
 
   if (is_null(in_place)) {
-    message("Not persisting, use `in_place = FALSE` to turn off this message.")
+    inform("Result is returned as lazy table. Use `in_place = FALSE` to mute this message, or `in_place = TRUE` to write to the underlying table.")
```
Comment (krlmlr) [inline R/rows-dm.R:209]:
```suggestion
    inform("Result is returned as a dm object with lazy tables. Use `in_place = FALSE` to mute this message, or `in_place = TRUE` to write to the underlying tables.")
```

**Review by krlmlr — COMMENTED** [review]
Thanks.

File: R/rows-dm.R (hunk 1 above)
Comment (IndrajeetPatil) [reply R/rows-dm.R]: Incorporated the suggested change and updated the snaps. Thanks.

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**